### ISSUE TITLE
Keymapping issue

### ISSUE THREAD

**Issue by za3k** [kernc/logkeys#245]
It looks like something odd is going on with keymaps. On two computers, I run:

     sudo logkeys -s -u --no-daemon

I add keycode logging by adding a printf in `update_key_state` at the top of the default case.

On both, I press the "1" button. On both computers, keycode 2 is registered, which matches what I expect from ex. https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h. On both, I see `/var/log/logkeys.log` log key `2` as being pressed (incorrect).

On both, I press the "q" button. On both computers, keycode 16 is registered.
One one computer, `w` is logged to the file. On the other, `q` is logged to the file.

In short, they don't seem to be using the same keymap, even though `-u` is passed.

**Comment by za3k** [comment]
Tracking it down further, the `-u` option is not used except together with `--export-keymap`. The code simply ignores it.
